website/docs/fb/fbcode.md

```diff
@@ -22,8 +22,8 @@ Hydra includes buck TARGETS you can use in fbcode. In general, if there is TARGE
 1. You can depend on the TARGETS to use Hydra or a plugin.
 2. The TARGETS contains a runnable example.
```

Comment:
should we also include what jack suggested? or maybe the examples are self-explanatory enough

```
 in order to use yamls in fbcode, you need to add them to your TARGETS resources: resources = glob(["**/*.yaml"]),
```

Reply:
1. I think we need links to the code tree for hydra, plugins and examples below.
2. We can also add a direct link to TARGETS of 6_composition, which should contain it. I don't think we need to spell it out, it should overall be familiar to fbcode users.